docs/helpers/custom_helpers.rb

```diff
@@ -0,0 +1,48 @@
+require 'yaml'
+
+module CustomHelpers
+
+  def create_navigation
+  	navigation = ""
+  	topic_map = YAML.load_file('build/_topic_map.yml')
```

Comment:
-1, that won't have the command ref links. They are getting added as part of the build and the file gets generated under _build/_topic_map.yml_.